apps/editor/__tests__/specs/__snapshots__/utility.test.ts.snap

```diff
@@ -54,7 +54,7 @@ exports[`slugify should transform text with special r like ŔŖŘ correctly 1`]
 
 exports[`slugify should transform text with special s like ŚŜŞŠ correctly 1`] = `"text-with-special-s-like-ssss"`;
 
-exports[`slugify should transform text with special ss like ß correctly 1`] = `"text-with-special-ss-like-ss"`;
+exports[`slugify should transform text with special ss like ß correctly 1`] = `"text-with-special-ss-like-s"`;
```

Comment:
should this be ss or s? ß should be ss technically. One slugify method did this correctly, the other didn't.